samples/bluetooth/nrf_dm/child_image/hci_rpmsg.conf

```diff
@@ -17,8 +17,12 @@ CONFIG_MPSL_TIMESLOT_SESSION_COUNT=1
 # Main Stack
 CONFIG_MAIN_STACK_SIZE=680
 
+# System workqueue
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+
 # BT
 CONFIG_BT_EXT_ADV=y
 
 # IPC
-CONFIG_IPC_SERVICE_BACKEND_RPMSG_NUM_ENDPOINTS_PER_INSTANCE=3
+CONFIG_IPC_SERVICE_BACKEND_ICMSG_ME_NUM_EP=3
```

Comment:
I think `IPC_SERVICE_BACKEND_ICMSG_ME_NUM_EP` could be merged together with `IPC_SERVICE_BACKEND_RPMSG_NUM_ENDPOINTS_PER_INSTANCE`. There is absolutely no reason why the same configuration from the user point of view should have different names depending on the backend.